projects/lendai-us/design-system/src/lib/steper/step-group/step-group.component.html

```diff
@@ -0,0 +1,13 @@
+<lai-step-group-header>
+  <lai-step-label
+    *ngFor="let step of steps; let index = index"
+    [step]="step"
+    [selected]="step === selectedStep"
```

Comment:
use some input field three options `previous`, `current`, `next` as we discussed (without dispatcher and related stuff that we discussed today)

this way instead of managing both `selected` property and `.active` class we will have only 1 property for step label to manage, which makes it simpler

Reply:
with the current solution, the state of the label is still managed in two different places:

- `.previous` and `.next` classes are set here
- `.current` is set inside `StepLabelComponent` based on the `selected` input

please change `selected` input from being a `boolean` type to be a string union (probably the name of the property should also be changed to something like `state`) of 3 values `previous`, `current` and `next`. then based on the index that we are getting from `ngFor` loop pass the right state to the label. and only then label itself can bind proper class to itself and adjust its stylesheets